### ISSUE TITLE
Empty dropdowns can be expanded in Edge and Internet Explorer

### ISSUE THREAD

**Issue by Naktibalda** [zenkiezhu/jquery-cascading-select#5]
This plugin looks and behaves perfectly on Firefox.
If Dropdown with no options is clicked, a few pixels wide darker line appears below dropdown.

Edge browser (v42) expands a large empty area with black border:

![image](https://user-images.githubusercontent.com/395992/46405786-03afed00-c712-11e8-8e6c-a234f0cf1416.png)

The same issue exists with Internet Explorer 11.

**Comment by uqix** [comment]
Is this IE's problem? maybe try the option `placeholderWhenEmpty`

**Comment by Naktibalda** [comment]
Thanks, placeholderWhenEmpty certainly helps - it makes IE/Edge display a single blank row in dropdown.

However `placeholderWhenEmpty` option is not applied on initial rendering of dropdowns, it is only applied after selecting a value in the first dropdown.

I investigated this issue and pinned it down to `preselect()` function.
My application is rendering options of the first dropdown in HTML, so `init()` calls `preselect()` and `preselect()` knows nothing about `placeholderWhenEmpty`.

I updated `preselect()` to handle `placeholderWhenEmpty` in the same way as `change` in my codebase.
I will make pull request in a moment.
